pythran/pythonic/include/numpy/around.hpp

```diff
@@ -2,29 +2,48 @@
 #define PYTHONIC_INCLUDE_NUMPY_AROUND_HPP
 
 #include "pythonic/include/numpy/rint.hpp"
-#include <nt2/include/functions/pow.hpp>
+#include "pythonic/include/numpy/asarray.hpp"
+#include "pythonic/include/numpy/sum.hpp"
 
 namespace pythonic
 {
 
   namespace numpy
   {
+    namespace details
+    {
+      template <class T>
+      struct is_integral_valued {
+        // what a trick :-) use sum to get the dtype even when T is a scalar
+        // type
```

Comment:
How do you now if a type, wich is a valid numexpr eargument, has an integral dtype.

there are two cases:
- it's a numpexpr or such, and in thatc ase it has a dtype
- it's a constant

The sum trick projects both types on a scalar, that's a funny trick